pydataapi/pydataapi.py

```diff
@@ -356,8 +366,12 @@ def __init__(
         transaction_id: Optional[str] = None,
         client: Optional[boto3.session.Session.client] = None,
         rollback_exception: Optional[Type[Exception]] = None,
+        rds_client: Optional[boto3.session.Session.client] = None,
     ) -> None:
-        self.resource_arn: str = resource_arn
+        if resource_arn.startswith('arn:aws:rds:'):
+            self.resource_arn: str = resource_arn
+        else:
+            self.resource_arn = find_arn_by_resource_name(resource_arn, rds_client)
```

Comment:
We set resource name to `resource_arn`,
However, I should think a better way 🤔

Reply:
I think we can either pass `resource_arn` or `resource_name` to `connect_args`. If we pass `resource_arn` then we do nothing. If we pass `resource_name` then we retireve arn.

Reply:
I just changed it.